lib/src/chart/base/axis_chart/axis_chart_data.dart

```diff
@@ -970,6 +971,10 @@ class HorizontalLine extends FlLine with EquatableMixin {
   /// Draws a text label over the line.
   final HorizontalLineLabel label;
 
+  /// if not drawing dash line, then this is the Strokecap for the line.
```

Comment:
Why you said `if not drawing dash line`?
I checked and it works well with dashed lines.

<img width="300" alt="image" src="https://user-images.githubusercontent.com/7009300/230940187-e3858b60-3bfd-4a48-b82c-a002ce3e87b5.png">